components/webext-storage/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "webext-storage"
+edition = "2018"
+version = "0.1.0"
+authors = ["application-services@mozilla.com"]
+license = "MPL-2.0"
+
+[features]
+log_query_plans = ["sql-support/log_query_plans"]
+default = []
+
+[dependencies]
+error-support = { path = "../support/error" }
+failure = "0.1.6"
+interrupt = { path = "../support/interrupt" }
+lazy_static = "1.4.0"
+log = "0.4"
+serde = "1"
+serde_json = "1"
+serde_derive = "1"
+sql-support = { path = "../support/sql" }
+sync15 = { path = "../sync15" }
+sync-guid = { path = "../support/guid", features = ["rusqlite_support", "random"] }
+url = { version = "2.1.1", features = ["serde"] }
+
+[dependencies.rusqlite]
+version = "0.21.0"
+features = ["functions", "bundled"]
```

Comment:
in_gecko will take priority over bundled (and essentially all other features controlling build configuration): https://github.com/jgallagher/rusqlite/blob/master/libsqlite3-sys/build.rs#L7

It's fine to use these features here, the way cargo features combine should make it so that because anyone (gkrust in this case) turns on the in_gecko feature, it's always on without this extra work you're describing for setting the desktop build flags